src/types/StreamingInfoOptions.ts

```diff
@@ -26,5 +26,5 @@ export interface StreamingInfoOptions {
    *
    * Defaults to `(audio_track_display_name) => audio_track_display_name + " (Stable Volume)"`
    */
-  label_drc_mutiple?: (audio_track_display_name: string) => string;
+  label_drc_multiple?: (audio_track_display_name: string) => string;
```

Comment:
^^ Technically, this is also a breaking change. Since this was going to be a big refactor anyway, I fixed some typos and removed deprecated code (like the YouTube Studio stuff).